src/Libraries/Microsoft.AspNetCore.HeaderParsing/HeaderParsingFeature.cs

```diff
@@ -198,12 +198,12 @@ private void CopyTo(Box<T> to)
         }
     }
 
-    [LoggerMessage(LogLevel.Debug, "Can't parse header '{HeaderName}' due to '{Error}'.")]
+    [LoggerMessage(LogLevel.Debug, "Can't parse header '{HeaderName}' due to '{Error}'.")] 
```

Comment:
While `LogParsingError` and `LogNotFound` messages log erroneous situations, making them of level >= Information would result in logging of a bunch of messages. So, I think it is good to leave them as is of Debug level, isn't it?

Reply:
Originally `LogParsingError` had level Warning, then it was changed to Debug.

@geeknoid Can you help here, please? Are we good to have log level for `LogParsingError` message as Debug, though it reports erroneous situation? Thank you.

Reply:
This must be debug. Otherwise, an attacker could set malformed headers and trigger a large amount of log spew from the service, which could potentially lead to an outage.